draft-ietf-quic-recovery.md

```diff
@@ -1141,6 +1141,10 @@ SetLossDetectionTimer():
     // Time threshold loss detection.
     loss_detection_timer.update(earliest_loss_time)
     return
+    
+  if (IsServerWhichReachedAddressValidationLimit()):
+    loss_detection_timer.cancel()
+    return
```

Comment:
The odd mix of pseudo-prose and undefined functions is starting to get noticeable.

```
    if (server is at anti-amplification limit):
```

And later:

```
    if (no ack-eliciting packets in flight &&
        peer not awaiting address validation):
```